thorlcr/graph/thgraph.hpp

```diff
@@ -630,7 +630,7 @@ class graph_decl CGraphBase : public CGraphStub, implements IEclGraphResults
     CChildGraphTable childGraphsTable;
     CGraphStubArrayCopy orderedChildGraphs;
     Owned<IGraphTempHandler> tmpHandler;
-    Owned<CFileSizeTracker> tempFileSizeTracker;
+    mutable OwnedSingleton<CFileSizeTracker> tempFileSizeTracker;
```

Comment:
not sure why it needs to now be mutable, it isn't used from a const context.

Reply:
it needed to be mutable, because queryActiveTempSize etc. are const, but Singleton::query wasn't.
I've changed SingletonBase::query to be const now.